hadoop-ozone/dist/src/main/k8s/examples/ozone-ha/scm-statefulset.yaml

```diff
@@ -52,6 +52,15 @@ spec:
         volumeMounts:
         - name: data
           mountPath: /data
+      - name: bootstrap
+        image: '@docker.image@'
+        args:
+        - ozone
+        - scm
+        - --bootstrap
```

Comment:
Do all three sccm instances neet to start with --bootstrap?

Reply:
no, the first scm instance will be started with --init, and the other two will be started will --bootstrap,

Reply:
@ChenSammi the logic here is as follows.
we want to create 3 scm replicas  in this yaml file. for each scm instance, two init containers will be executed first, and then the container will be started. it seem like we have 3 individual scm , and for each scm, we do `ozone scm --init`, `ozone scm --bootstrap` and `ozone scm` sequentially one by one. 

so for the first(primordial) scm, `ozone scm --init` succeeds , and  the next `ozone scm --bootstrap` will fail with the message (SCM bootstrap command can only be executed in non-Primordial SCM), then it is started by `ozone scm` as a primordial scm.

for the second and third scm, `ozone scm --init` will fail with the message(SCM init command can only be executed in Primordial SCM), and `ozone scm --bootstrap` will succeed, and then it is started by `ozone scm` as a non-primordial scm

i am not sure is it clear to you ? please let me know if the description is no clear, thanks